cmd/terramate/cli/cli.go

```diff
@@ -513,14 +675,27 @@ func (c *cli) printStacksGlobals() error {
 }
 
 func (c *cli) printMetadata() error {
+	logger := log.With().
+		Str("action", "printMetadata()").
+		Logger()
+
+	logger.Trace().
+		Str("stack", c.wd()).
+		Msg("Load metadata.")
 	metadata, err := terramate.LoadMetadata(c.root())
 	if err != nil {
 		return err
 	}
 
+	logger.Trace().
+		Str("stack", c.wd()).
```

Comment:
```suggestion
		Str("workingDir", c.wd()).
```
+ it could be extracted to the logger.